cmd/lk/cloud.go

```diff
@@ -269,15 +269,21 @@ func tryAuthIfNeeded(ctx context.Context, cmd *cli.Command) error {
 		return err
 	}
 
-	// name
-	var deviceName string
+	// get devicename
+	deviceName := cliConfig.DeviceName
 	if err := huh.NewInput().
 		Title("What is the name of this device?").
 		Value(&deviceName).
```

Comment:
Nit: you can just pass in `&cliConfig.DeviceName` here and skip the intermediate string allocation

Reply:
great tip, thanks!